### ISSUE TITLE
BUG: model.close and applicaton.close dont work

### ISSUE THREAD

**Issue by talSas** [Dlubal-Software/Dlubal_CSharp_Client#5]
**Describe the bug**
When calling the close method nothing happens
**To Reproduce**

1. Create API Connection to a RSTAB Model
2. Call model.close or application.close
3.  see error  or in this case nothing
**Expected behavior**
I ex

**Screenshots**
i expect the model to close when i call model.close and the application to close when i call application.close

**Desktop (please complete the following information):**

    OS: e.g. WIN 10 PRO Version 10.0.19043 Build 19043
    RSTAB 9.01.0008

**Additional context**

**Comment by jarabroz** [comment]
Hi @talSas,
I have checked this issue and I'm not able to reproduce it. I use public void close_model(int index, bool save_changes) for closing of model and  public void close_application() for closing of application. Please can you try to use the RSTAB 9.01.009?

**Comment by talSas** [comment]
Hey,
yes you are right with the methods RstabApplicationClient.close_application() and 

RstabApplicationClient.close_model(int no, bool saveChanges) 

the closing of the application and the model works.

Besides these methods there are RstabModelClient.Close()

and a simple RstabApplicationClient.Close()

these last two methods have no function, respectively no recognizable one.

Not even in RSTAB 9.01.009.

Best regards

**Comment by jarabroz** [comment]
Ah, I did not know them. They are from parent class ClientBase. See documentation here https://docs.microsoft.com/en-us/dotnet/api/system.servicemodel.clientbase-1?view=dotnet-plat-ext-6.0.
Close() | Causes the ClientBase<TChannel> object to transition from its current state into the closed state. If I got it right, you close connection with Server - disconnect from model in RSTAB or application itself. Once used you will need to reconnect.

**Comment by jarabroz** [comment]
Ah, I did not know them. They are from parent class ClientBase. See documentation here https://docs.microsoft.com/en-us/dotnet/api/system.servicemodel.clientbase-1?view=dotnet-plat-ext-6.0.
Close() | Causes the ClientBase<TChannel> object to transition from its current state into the closed state. If I got it right, you close connection with Server - disconnect from model in RSTAB or application itself. Once used you will need to reconnect.
